src/wallet/test/fuzz/coinselection.cpp

```diff
@@ -45,6 +45,37 @@ static void GroupCoins(FuzzedDataProvider& fuzzed_data_provider, const std::vect
     if (valid_outputgroup) output_groups.push_back(output_group);
 }
 
+static CAmount CreateCoins(FuzzedDataProvider& fuzzed_data_provider, std::vector<COutput>& utxo_pool, CoinSelectionParams& coin_params)
+{
+    // Create some coins
+    CAmount total_balance{0};
+    int next_locktime{0};
```

Comment:
Need to make `next_locktime` a global variable (or receive it as ref here).

This variable prevents `CreateCoins` from generating duplicated transaction hashes (look at `AddCoin()`). If the variable is reset on every `CreateCoins`, as is now, could be the case where the same coins are created twice.
Which will cause a runtime_error in `AddInputs()` and `Merge()` calls (see `SelectionResult::InsertInputs()`).

Reply:
Done! Thanks!